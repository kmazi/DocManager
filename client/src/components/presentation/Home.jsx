import React from 'react';

import Auth from '../container/Auth';

/**
 * Renders the home screen of the application
 * @return {object} - Returns the html to render
 */
const home = () => (
  <div className="homescreen">
    <h1 className="center-align">DocManger</h1>
    <p className="center-align">
      Create, secure and access documents
      from anywhere in the world.
    </p>
    <Auth />
  </div>
  );
export default home;
