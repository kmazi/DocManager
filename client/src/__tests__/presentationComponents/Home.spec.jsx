import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '../../components/presentation/Home';

describe('The Home component:', () => {
  const enzymeWrapper = shallow(<Home />);
  test('should render the Home component correctly', () => {
    const component = enzymeWrapper;
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
