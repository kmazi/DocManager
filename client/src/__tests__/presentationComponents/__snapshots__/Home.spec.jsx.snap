// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home component: should render the Home component correctly 1`] = `
<div
  className="row homescreen"
>
  <h1
    className="center-align"
  >
    DocManger
  </h1>
  <p
    className="center-align"
  >
    Create, secure and access documents from anywhere in the world.
  </p>
  <Connect(withRouter(Authenticate)) />
</div>
`;
